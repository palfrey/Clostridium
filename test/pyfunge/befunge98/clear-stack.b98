123n#v_#v_#v_"kcats eht sraelc n :DOOG",,,,,,,,,,,,,,,,,,,,,,,,@
     >  >  >"kcats eht raelc ton seod n :DAB",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,@
