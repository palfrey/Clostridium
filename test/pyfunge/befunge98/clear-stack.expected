GOOD: n clears the stack
